draft-ietf-quic-transport.md

```diff
@@ -182,7 +188,15 @@ QUIC packet:
 
 : A well-formed UDP payload that can be parsed by a QUIC receiver.
 
-QUIC is a name, not an acronym.
+0-RTT:
+
+: QUIC packets containing application payload data that are encrypted with
```

Comment:
Technically 0-RTT packets contain application data and some CRYPTO data, so I'd be inclined to just say "QUIC packets encrypted with a key derived from a previous QUIC connection to the same endpoint."